float32[] joint_angles    # NxD joint angle matrix
float64[] times           # N timestamps (in seconds)
